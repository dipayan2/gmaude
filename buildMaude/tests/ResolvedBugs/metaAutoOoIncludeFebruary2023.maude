set show timing off .

***
***	Test that auto object-oriented includes show up in the unflattened result of upModule().
***

oth FOO is
endoth

red in META-LEVEL : upModule('FOO, false) .
