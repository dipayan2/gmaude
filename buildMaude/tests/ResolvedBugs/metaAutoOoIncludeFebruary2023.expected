==========================================
reduce in META-LEVEL : upModule('FOO, false) .
rewrites: 1
result STheory: th 'FOO is
  including 'CONFIGURATION .
  sorts none .
  none
  none
  none
  none
  none
endth
Bye.
